### ISSUE TITLE
Mojave Camera Privacy 

### ISSUE THREAD

**Issue by jonaylor89** [jwilm/alacritty#1733]
MacOS Mojave has a new privacy thing that won't allow terminals to run applications needing the camera. I wrote something in OpenCV and here is a piece the crash report.

```
Process:               Python [21714]
Path:                  /usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/Resources/Python.app/Contents/MacOS/Python
Identifier:            Python
Version:               3.7.0 (3.7.0)
Code Type:             X86-64 (Native)
Parent Process:        zsh [19334]
Responsible:           Python [21714]
User ID:               501

Date/Time:             2018-10-31 12:14:07.256 -0400
OS Version:            Mac OS X 10.14 (18A391)
Report Version:        12
Anonymous UUID:        F0A7A4BD-376E-8ABB-C1A9-17460E44D001

Sleep/Wake UUID:       4007470B-5958-4108-8D7E-BCC88E7BE11D

Time Awake Since Boot: 35000 seconds
Time Since Wake:       3700 seconds

System Integrity Protection: enabled

Crashed Thread:        1  Dispatch queue: com.apple.root.default-qos

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    Namespace TCC, Code 0x0
```
iTerm also has this issue and it seems that the only terminal emulator that doesn't have the problem is the native Terminal.app. I believe something just needs to be added to the Info.plist to allow the camera access but I'm not entirely sure. 

(This is my first time reporting an issue to an open-source project so if there is someway I could improve it please let me know)

**Comment by chrisduerr** [comment]
> (This is my first time reporting an issue to an open-source projust so if there is someway I could improve it please let me know)

Thanks for reporting the problem! It's a very specific issue with a recent macOS version so it might take some time until someone gets to it though.

**Comment by chrisduerr** [comment]
Fixed in #1760.
